docs/UserGuide.md

```diff
@@ -334,18 +364,18 @@ Should you want to re-enter your contacts in a fresh JSON file in the event of f
 simply delete `realodex.json` and restart the app. A new JSON file with sample contacts will be generated and you may proceed from there.
 
 --------------------------------------------------------------------------------------------------------------------
-## Format Requirements
+## Field Constraints
```

Comment:
If you rename this to Field Constraints, need to rename format-requirements to field-constraints in default.md also